src/main/java/com/googlecode/lanterna/gui2/Window.java

```diff
@@ -180,6 +181,13 @@ public String toString() {
     @Override
     void invalidate();
 
+    
+    default TerminalRectangle getBounds() {
```

Comment:
Need to add Javadoc to this one and the two below

Reply:
I've updated with some javadocs.